suse/rdma-core.spec

```diff
@@ -0,0 +1,698 @@
+BuildRequires:  pkgconfig(libnl-3.0)
+BuildRequires:  pkgconfig(libnl-route-3.0)
+BuildRequires:  pkgconfig(systemd)
+Requires:       dracut
+Requires:       kmod
+Requires:       systemd
+
+# SUSE previously shipped rdma as a stand-alone
+# package which we're supplanting here.
+
+Provides:       rdma = %{version}
+Obsoletes:      rdma < %{version}
+Provides:       ofed = %{version}
+Obsoletes:      ofed < %{version}
+
+%if 0%{?suse_version} >= 1330
+BuildRequires:  curl-mini
+%endif
+
+# Tumbleweed's cmake RPM macro adds -Wl,--no-undefined to the module flags
+# which is totally inappropriate and breaks building 'ENABLE_EXPORTS' style
+# module libraries (eg ibacmp).
+#%%define CMAKE_FLAGS -DCMAKE_MODULE_LINKER_FLAGS=""
+
+# Since we recommend developers use Ninja, so should packagers, for consistency.
+%define CMAKE_FLAGS %{nil}
+%if 0%{?suse_version} >= 1300
+BuildRequires:  ninja
+%define CMAKE_FLAGS -GNinja
+%define make_jobs ninja -v %{?_smp_mflags}
+%define cmake_install DESTDIR=%{buildroot} ninja install
+%else
+# Fallback to make otherwise
+BuildRequires:  make
+%define make_jobs make -v %{?_smp_mflags}
+%define cmake_install DESTDIR=%{buildroot} make install
+%endif
+
+%description
+RDMA core userspace infrastructure and documentation, including initialization
+scripts, kernel driver-specific modprobe override configs, IPoIB network
+scripts, dracut rules, and the rdma-ndd utility.
+
+%package devel
+Summary:        RDMA core development libraries and headers
+Group:          Development/Libraries/C and C++
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+Requires:       %{ibcm_lname} = %{version}-%{release}
+Requires:       %{rdmacm_lname} = %{version}-%{release}
+Requires:       %{umad_lname} = %{version}-%{release}
+Requires:       %{verbs_lname} = %{version}-%{release}
+%%if 0%{?dma_coherent}
+Requires:       %{mlx4_lname} = %{version}-%{release}
+Requires:       %{mlx5_lname} = %{version}-%{release}
+%endif
+Requires:       rsocket = %{version}-%{release}
+
+Provides:       libibverbs-devel = %{version}-%{release}
+Obsoletes:      libibverbs-devel < %{version}-%{release}
+
+Provides:       libibcm-devel = %{version}-%{release}
+Obsoletes:      libibcm-devel < %{version}-%{release}
+
+Provides:       libibumad-devel = %{version}-%{release}
+Obsoletes:      libibumad-devel < %{version}-%{release}
+Provides:       librdmacm-devel = %{version}-%{release}
+
+Obsoletes:      librdmacm-devel < %{version}-%{release}
+#Requires:       ibacm = %%{version}-%%{release}
+Provides:       ibacm-devel = %{version}-%{release}
+Obsoletes:      ibacm-devel < %{version}-%{release}
+
+%description devel
+RDMA core development libraries and headers.
+
+%package -n     libibverbs
+Summary:        Library & drivers for direct userspace use of InfiniBand/iWARP/RoCE hardware
+Group:          System/Libraries
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Provides:       libcxgb3-rdmav2 = %{version}-%{release}
```

Comment:
@nmorey I don't know much about packaging, this line "Provides:       libcxgb3-rdmav2" and the word "provides" worried me.